src/app/games/tafl/tests/MyTaflRules.spec.ts

```diff
@@ -16,7 +16,7 @@ export class MyTaflRules extends TaflRules<MyTaflMove> {
         new RulesConfigDescription<TaflConfig>({
             name: (): string => `MyTafl`,
             config: {
-                castleIsLeftForGood: new BooleanConfig(true, () => $localize`Central throne is left for good`),
+                canReturnInCastle: new BooleanConfig(false, () => $localize`Central throne is left for good`),
```

Comment:
castle ou throne ?
canReturn TO castle non ?

Reply:
"To ou In" comme tu veux, par contre je dirais que throne = les 5, while castle serait uniquement celui du centre ?

Reply:
can return to castle ouais
et va pour castle = centre, throne = bords.
Tu pars du chateau pour aller sur ton trone et gagner